README.md

```diff
@@ -86,24 +93,56 @@ OPTIONS:
 
 # Docker
 
-To run `tofnd` inside a container, run:
+## Setup
+To setup a `tofnd` container, use the `create` mnemonic command:
 
 ```
-docker-compose up
+docker-compose run -e MNEMONIC_CMD=create tofnd
 ```
 
-For testing purposes, `docker-compose.unsafe.yml` is available which is equivelent to `cargo run -- --unsafe`. To create an unsafe `tofnd` container, run
+This will initialize `tofnd`, and then exit.
+
+## Execution
+
+To run a `tofnd` daemon inside a container, run:
 
 ```
-docker-compose -f docker-compose.unsafe.yml up
+docker-compose up
 ```
 
+## Storage
+
 We use [data containers](https://docs.docker.com/engine/reference/commandline/volume_create/) to persist data across restarts. To clean up storage, remove all `tofnd` containers, and run
 
 ```
 docker volume rm tofnd_tofnd
 ```
 
+## Testing
+
+For testing purposes, `docker-compose.test.yml` is available, which is equivelent to `./tofnd --no-password --unsafe`. To spin up a test `tofnd` container, run
+
+```
+docker-compose -f docker-compose.test.yml up
+```
+
+## The `auto` command
+
+In containerized environments, the `auto` mnemonic command can be used. `auto` works as follows:
+1. If no `import` file exists in `TOFND_HOME`, then `tofnd` attempts to create a new mnemonic. 
+    1. If an mnemonic is already stored in the kv store, then the creation is aborted, and the existing mnemonic is used.
+    2. If no mnemonic is stored in the kv store, then a fresh mnemonic is created, and the mnemonic is exported. The exported file is then renamed to `TOFND_HOME/import`.
+2. If an `import` file exists in `TOFND_HOME`, then `tofnd` attempts to use it. 
+    1. If a mnemonic is already stored in the kv store, then the import is aborted, and the existing mnemonic is used.
+    2. If no mnemonic is stored in the kv store, then the `import` file is used to store the mnemonic in the kv store.
+
+In short, `auto` implies the following order: 1) try to use existing mnemonic, 2) try to import a mnemonic from file, 3) try create a new mnemonic.
```

Comment:
```suggestion
In containerized environments the `auto` mnemonic command can be used.  This command is implemented in `entrypoint.sh` and does the following:
1. Try to use existing mnemonic.  If successful then launch `tofnd` server.
2. Try to import a mnemonic from file.  If successful then launch `tofnd` server.
3. Create a new mnemonic.  The newly created mnemonic is automatically written to the file `TOFND_HOME/export`---rename this file to `TOFND_HOME/import` so as to unblock future executions of tofnd.  Then launch `tofnd` server.
```

These instructions are pretty complicated.  How about this instead?  It's unfortunate that `entrypoint.sh` doesn't actually act exactly as described above but the effect is the same.  I don't think it's worth describing what `entrypoint.sh` actually does.  But I do wonder whether it's worth changing `entrypoint.sh` to somehow behave exactly as this suggestion describes.

Reply:
Yeah I guess I got carried away. I like narrowing down to simpler steps. BTW, by changing the order to 2 -> 3 -> 1 you describe exactly how `entrypoint.sh` uses `auto`.